src/java.desktop/share/native/libharfbuzz/hb-coretext.cc

```diff
@@ -1,1196 +0,0 @@
-/*
```

Comment:
How we will track these files for the next HB update? Do we need to document it somehow? otherwise, I think it will be added again as a new files.

Reply:
No, they won't. We already pull in only the files we need, not every file, so this is not a precedent.